tools/cmake/Makefile

```diff
@@ -7,14 +7,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cmake
-PKG_VERSION:=3.19.1
-PKG_RELEASE:=1
```

Comment:
Please just keep it, I don't think it harms and it's better to leave it rather than adding this line every now and then.

Reply:
Done.